### PR TITLE
Update README.md for next version

### PR DESCRIPTION
**Pull request by jakaplan**
Once #64 and #65 have been merged in, this can be merged and then the next version released

### CODE REVIEW

**Comment by jakaplan** [comment]
@amomchilov your feedback is most welcome

File: README.md
Comment (amomchilov) [inline README.md:1]:
```suggestion
Use pure Swift to easily and securely communicate with XPC and Mach services, with customized support for
```

File: README.md
Comment (amomchilov) [inline README.md:2]: Seems a bit "too early" to mention this.

I liked your original phrasing:

> This framework is ideal for communicating with helper tools installed via 
[`SMJobBless`](https://developer.apple.com/documentation/servicemanagement/1431078-smjobbless).

File: README.md
Comment (amomchilov) [inline README.md:4]: Active voice:

> SecureXPC uses a client-server model...


Or perhaps:

> SecureXPC follows a client-server model, using Codable to encode messages and their replies.

File: README.md
Comment (amomchilov) [inline README.md:7]: I would invert this to focus on the newer. It's novel, attention-grabbing and awesome.

> SecureXPC uses [Swift Concurrency](https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html) to allow clients to make non-blocking, asynchronous calls to servers. A closure-based API is also available, for backwards compatibility for macOS versions prior to 10.15.

**Review by amomchilov — APPROVED** [review]
I would draw more attention to the parts of the framework that really standout, which IMO are:

1. Swift-first, integrating with user defined types, Codable, etc.
2. Support for Swift concurrency (which is huge, IMO, and quite a "hot topic")
3. Secure (linked out to some explanations about the risks of unprotected privileged helpers)

**Review by amomchilov — APPROVED** [review]
Excellent revision, this is great!

**Comment by jakaplan** [comment]
> Excellent revision, this is great!

Thanks for the feedback. Will merge this after #64 has been merged, then make an official release.

**Pull request merged**
